src/main/resources/DB/init.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE Homework
+(
+id 		integer GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+name 	VARCHAR(40) NOT NULL,
+description VARCHAR(200)
+);
+CREATE TABLE Lesson
+(
+id		integer GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+name	VARCHAR(40) NOT NULL,
+updatedAt TIMESTAMP DEFAULT NOW(),
```

Comment:
одразу не побачив: в базах даних колонки через підкреслювання називаються 
```suggestion
updated_at TIMESTAMP DEFAULT NOW(),
```